int2lm/Dockerfile

```diff
@@ -0,0 +1,44 @@
+# Build stage with Spack pre-installed and ready to be used
+FROM c2sm/nvhpc:21.3-devel-cuda_multi-ubuntu20.04-mpich
+ENV INT2LM_SPEC="int2lm@c2sm-master%nvhpc ^cmake%gcc"
```

Comment:
Does Spack choose mpich as MPI?
I think the default is OpenMPI.

The image providing mpich provides an env-variable:
```
ENV COSMO_SPEC="cosmo@c2sm-master%nvhpc cosmo_target=cpu cuda_arch=60 ~cppdycore ^$MPICH_SPEC"
```
that we should use here as well.